.github/workflows/examples-telink.yaml

```diff
@@ -57,12 +57,27 @@ jobs:
             - name: Build example Telink Lighting App
               run: |
                   ./scripts/run_in_build_env.sh \
-                    "./scripts/build/build_examples.py --no-log-timestamps --target-glob 'telink-*' build"
+                    "./scripts/build/build_examples.py --no-log-timestamps --target-glob 'telink-tlsr9518adk80d-light' build"
```

Comment:
no need for a glob if you are not using glob features like `*` or `?`. Just use `--target`

Reply:
Fixed